integration/e2e/step-definitions/zoom.steps.ts

```diff
@@ -0,0 +1,69 @@
+import { ViewerPage } from '../pages/viewer.po';
+import { defineSupportCode } from 'cucumber';
+import { expect } from '../helpers/chai-imports';
+
+defineSupportCode(function ({ Given, When, Then }) {
+  const page = new ViewerPage();
+  let previousZoomLevel = 0;
+
+  Given(/^zoom level is home$/, async () => {
+    await page.setHomeZoom();
+    await page.waitForAnimation();
+  });
+
+  Given(/^the view is all zoomed out$/, async () => {
+    await page.getMinZoom().then((zoomLevel: number) => page.setZoomLevel(zoomLevel));
```

Comment:
This could be done like this:

const zoomLevel = await page.getMinZoom();
page.setZoomLevel(zoomLevel));